R/FilteredData.R

```diff
@@ -901,6 +934,10 @@ FilteredData <- R6::R6Class( # nolint
     # keys used for joining/filtering data named nested lists
     keys = NULL,
 
+    # reactive i.e. filtered data
+    reactive_data = list(),
+    eval_env = list(),
```

Comment:
eval_env made sense in FilteredDataset where each Datasets had (reactive) env where parent was linked. I'd remove eval_env